recipe/patches/0002-Add-CMake-option-INSTALL_DEPENDENCY_DLLS-default-off.patch

```diff
@@ -1,23 +1,9 @@
-From 89d6caaf64732e0d76ae07ca3aae1b2c1335d54c Mon Sep 17 00:00:00 2001
```

Comment:
@nehaljwani do you mind taking a look at this patch?

Reply:
Looks like there might be [an option in CMake to this effect]( https://gitlab.com/graphviz/graphviz/-/blob/ee364e62be2ada0dd6b6e45f53e79e48b7c59382/CMakeLists.txt#L19 ). Though agree it would be good to hear from Nehal :)

Reply:
That option allows end-user to choose whether or not to use the libraries vendor-ed into the source distribution. The option that I added allows end-user to choose whether or not to install the found libraries on Windows via the install target. I should probably try to upstream my patch.